roles/vernemq/templates/z_vernemq_cluster.yaml.j2

```diff
@@ -28,6 +28,8 @@ spec:
       value: "1500000"
     - name: VERNEMQ_CONF
       value: "bWV0YWRhdGFfcGx1Z2luID0gdm1xX3N3YwpsaXN0ZW5lci52bXEuY2x1c3RlcmluZyA9ICRNWV9QT0RfSVA6NDQwNTMKbGlzdGVuZXIuaHR0cC5kZWZhdWx0ID0gMC4wLjAuMDo4ODg4CnBsdWdpbnMudm1xX3Bhc3N3ZCA9IG9mZgpwbHVnaW5zLnZtcV9hY2wgPSBvZmYKcGx1Z2lucy52bXFfazhzLnBhdGggPSAvdmVybmVtcS9wbHVnaW5zL19idWlsZC9kZWZhdWx0CnBsdWdpbnMuZG9qb3RfYWNsX3BsdWdpbi5wYXRoID0gL3Zlcm5lbXEvZG9qb3RfYWNsX3BsdWdpbi9kZWZhdWx0CnBsdWdpbnMuZG9qb3RfZGlzY29ubmVjdF9wbHVnaW4ucGF0aCA9IC92ZXJuZW1xL2Rvam90X2Rpc2Nvbm5lY3RfcGx1Z2luL2RlZmF1bHQKcGx1Z2lucy52bXFfazhzID0gb24KcGx1Z2lucy5kb2pvdF9hY2xfcGx1Z2luID0gb24KcGx1Z2lucy5kb2pvdF9kaXNjb25uZWN0X3BsdWdpbiA9IG9uCnBsdWdpbnMudm1xX2s4cy5wcmlvcml0eSA9IDMKcGx1Z2lucy5kb2pvdF9hY2xfcGx1Z2luLnByaW9yaXR5ID0gMgpwbHVnaW5zLmRvam90X2Rpc2Nvbm5lY3RfcGx1Z2luLnByaW9yaXR5ID0gMgpsb2cuY29uc29sZSA9IGNvbnNvbGUKbG9nLmNvbnNvbGUubGV2ZWwgPSBpbmZvCmxpc3RlbmVyLnNzbC5kZWZhdWx0ID0gMC4wLjAuMDo4ODgzCmxpc3RlbmVyLnNzbC5kZWZhdWx0LmNhZmlsZSA9ICAvdmVybmVtcS9jZXJ0L2NhYnVuZGxlLmNydApsaXN0ZW5lci5zc2wuZGVmYXVsdC5jZXJ0ZmlsZSA9IC92ZXJuZW1xL2NlcnQvaW90YWdlbnQtbXF0dC5jcnQKbGlzdGVuZXIuc3NsLmRlZmF1bHQua2V5ZmlsZSA9IC92ZXJuZW1xL2NlcnQvaW90YWdlbnQtbXF0dC5rZXkKbGlzdGVuZXIuc3NsLmRlZmF1bHQuY3JsZmlsZSA9ICAvdmVybmVtcS9jZXJ0L2NhLmNybApsaXN0ZW5lci5zc2wuZGVmYXVsdC51c2VfaWRlbnRpdHlfYXNfdXNlcm5hbWUgPSBvbgpsaXN0ZW5lci5zc2wuZGVmYXVsdC5yZXF1aXJlX2NlcnRpZmljYXRlID0gb24KbGlzdGVuZXIubWF4X2Nvbm5lY3Rpb25zID0gaW5maW5pdHkKZXJsYW5nLm1heF9wb3J0cyA9IDEzNDIxNzcyNwpzeXN0cmVlX2VuYWJsZWQgPSBvZmYKbGV2ZWxkYi5tYXhpbXVtX21lbW9yeS5wZXJjZW50ID0gMjAKbWF4X29ubGluZV9tZXNzYWdlcyA9IDUwMDAwCm1heF9vZmZsaW5lX21lc3NhZ2VzID0gNTAwMDAKbWF4X2luZmxpZ2h0X21lc3NhZ2VzID0gMTAwCnNoYXJlZF9zdWJzY3JpcHRpb25fcG9saWN5ID0gcHJlZmVyX2xvY2FsCmVybGFuZy5kaXN0cmlidXRpb25fYnVmZmVyX3NpemUgPSA1MTIwMDAKZXJsYW5nLmFzeW5jX3RocmVhZHMuc3RhY2tfc2l6ZSA9IDEzMTA3Mg=="
+    - name: REDIS_HOST
```

Comment:
probably with the changes in dojot/dojot#2136 and dojot/dojot#2135 a redis_host variable will not be exchanged for another one that will have the certificate_acl url